src/sourmash/save_load.py

```diff
@@ -214,6 +215,11 @@ def _load_sqlite_db(filename, **kwargs):
     return load_sqlite_index(filename)
 
 
+@add_loader("load collection from rocksdb", 20)
+def _load_sqlite_db(filename, **kwargs):
```

Comment:
```suggestion
def _load_disk_revindex(filename, **kwargs):
```

Is this overriding the `_load_sqlite_db` above? or the decorator renames the function?

Reply:
not overriding - you're right, it must rename the function!